.vscode/settings.json

```diff
@@ -28,6 +28,7 @@
     "**/*.snap",
     "**/*.story.tsx"
   ],
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Would you remove this? I prefer not to be forced to use one persons preferred formatter.